linux_os/guide/system/auditing/grub2_audit_backlog_limit_argument/rule.yml

```diff
@@ -23,6 +23,9 @@ identifiers:
     cce@rhel7: 82156-1
     cce@rhel8: 80943-4
 
+references:
+    srg: SRG-OS-000254-GPOS-00095
```

Comment:
Nack. ``SRG-OS-000254-GPOS-00095`` is for "operating system must initiate session audits at system start up"

Reply:
This rule helps ensure that auditing happens properly at system startup by properly setting a buffer size.  From the rule: "Until the audit daemon is up and running, all log messages are stored in this queue.  If the queue is overrun during boot process, the action defined by audit failure flag is taken."

Let me know if you agree and I can change if you don't

Reply:
Agreed.